wiki/Configuration:-Layer-Rules.md

```diff
@@ -79,7 +79,7 @@ They will be replaced with solid black rectangles.
 This can be useful for notifications.
 
 The same caveats and instructions apply as for the `block-out-from` window rule.
-Please read the `block-out-from` section in the [window rules](./Configuration:-Window-Rules.md) wiki page for more details.
+Please read the `block-out-from` section in the [window rules](./Configuration:-Window-Rules.md#block-out-from) wiki page for more details.
```

Comment:
With working anchors, you can move the link to "`block-out-from` section". The only reason it was on "window rules" is the missing anchor.

Similarly in other places.

Reply:
I'm out and about right now but I'll go through and fix some stuff like this later!